core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/util/base/RightV4.java

```diff
@@ -17,6 +17,8 @@
     DH_START(AuthorizationResourceAction.START_DATAHUB),
     DH_STOP(AuthorizationResourceAction.STOP_DATAHUB),
     DH_DELETE(AuthorizationResourceAction.DELETE_DATAHUB),
+    DL_UPGRADE(AuthorizationResourceAction.UPGRADE_DATALAKE),
```

Comment:
Can you please rename it to SDX_UPGRADE to match the others?

Reply:
done